src/go/rdctl/pkg/config/config.go

```diff
@@ -117,3 +133,24 @@ func finishConnectionSettings() (error, bool) {
 func insufficientConnectionInfo() bool {
 	return connectionSettings.Port == "" || connectionSettings.User == "" || connectionSettings.Password == ""
 }
+
+func getUserProfile() (string, error) {
+	var outBuf bytes.Buffer
+	cmd := exec.Command("cmd.exe", "/c", "echo %USERPROFILE%")
```

Comment:
I wonder how that interacts with Windows code pages (notably, CP932 / Japanese, where the backslash is ￥instead).

Reply:
How about this:

```console
jan@DESKTOP-4992680:/bin$ cmd.exe /c 'chcp 65001 >nul & echo %APPDATA%' 2>/dev/null
C:\Users\Jan\AppData\Roaming
```

The `&` is just a command separator for `cmd.exe` (like `;` for bash). Codepage 65001 is for UTF-8. The `>nul` is required to swallow the "Active code page: 65001" output of `dhcp`, and the final `2>/dev/null` is needed when you run the command from a directory that is not mapped back to Windows, e.g.

```console
jan@DESKTOP-4992680:/bin$ cmd.exe /c 'chcp 65001 >nul & echo %APPDATA%'
'\\wsl$\Ubuntu-20.04\usr\bin'
CMD.EXE was started with the above path as the current directory.
UNC paths are not supported.  Defaulting to Windows directory.
C:\Users\Jan\AppData\Roaming
```